service/frontend/redirection_interceptor.go

```diff
@@ -197,14 +202,14 @@ func (i *RedirectionInterceptor) handleLocalAPIInvocation(
 
 func (i *RedirectionInterceptor) handleRedirectAPIInvocation(
 	ctx context.Context,
-	req interface{},
+	req any,
 	info *grpc.UnaryServerInfo,
 	handler grpc.UnaryHandler,
 	methodName string,
-	respAllocFn func() interface{},
+	respCtorFn responseConstructorFn,
```

Comment:
what is Ctor? constructor?

Reply:
This is reflection of my nostalgic memories from the past years: https://stackoverflow.com/questions/4614099/what-is-the-meaning-of-ctor.